example/alertdog.yml

```diff
@@ -0,0 +1,45 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: alertdog
+  labels:
+    app: alertdog
+spec:
+  selector:
+    app: alertdog
+  ports:
+    - port: 9796
+      name: webhook
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: alertdog
+  labels:
+    app: alertdog
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: alertdog
+  template:
+    metadata:
+      labels:
+        app: alertdog
+    spec:
+      containers:
+      - name: alertdog
+        image: quay.io/errm/alertdog:pr-container-updates
```

Comment:
should probably point this at a release tag at some point

Reply:
Yes I should!

The first release was actually broken (wouldn't push to pagerduty bc missing ca-certs) ... so I need to make a new release!